repo/blob/azure/azure_storage.go

```diff
@@ -0,0 +1,273 @@
+// Package azure implements Azure Blob Storage.
+package azure
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/efarrer/iothrottler"
+	"github.com/pkg/errors"
+	"gocloud.dev/gcerrors"
+
+	"github.com/kopia/kopia/internal/retry"
+	"github.com/kopia/kopia/repo/blob"
+
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	gblob "gocloud.dev/blob"
+	"gocloud.dev/blob/azureblob"
+)
+
+const (
+	azStorageType   = "azure"
```

Comment:
technically, this is _azure blob_, not sure if the type should be `"azureBlob"` or `"azureblob"`

Reply:
Makes sense